ui/raidboss/data/07-dt/trial/valigarmanda-ex.txt

```diff
@@ -0,0 +1,178 @@
+# -r MjxcGWaA73mF8B1J -rf=104
+# -ii 8FF3 8FF5 8FF6 8FF7 8FF8 8FF9 8FE4 8FE6 8FCB 9013 9014 9015 901A 900D 900E 900F 9012 9016 8FC7 8FD9 87A2 9002 9003 9006 6854 9565 9566 9567 95B3 8FF4 8FDE 8FDD 8FDF 8FD7 8FD5
+# -p 9008:1006.7 95C3:2005.7 8FD1:3005.7 9007:4003.7
+# -it "Valigarmanda" "Flame-kissed Beacon"
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--sync--" InCombat { inGameCombat: "1" } window 0,1
+6.0 "--sync--" Ability { id: "8FF1", source: "Valigarmanda" }
+18.0 "Spikesicle" Ability { id: "8FF[24]", source: "Valigarmanda" } duration 11.8
```

Comment:
shouldn't id be `id: "8FF2|8ff4"` to be more inline with the rest where we have multiple ids?